spec/controllers/curation_concerns/generic_files_controller_spec.rb

```diff
@@ -72,7 +72,8 @@
       end
 
       context "when the file has a virus" do
-        xit "displays a flash error" do
+        it "displays a flash error" do
+          skip "pending hydra-works#89"
```

Comment:
:+1: to annotating why a spec is being skipped. This was a big headache in the initial code shredding project.